CMakeLists.txt

```diff
@@ -9,7 +9,16 @@ project(target-isns "C")
 set(TARGET_ISNS_VERSION "0.6.1")
 
 cmake_minimum_required(VERSION 2.8)
+
+set(IS_CMAKE_31 false)
+if(NOT CMAKE_VERSION VERSION_LESS "3.1")
+  set(IS_CMAKE_31 true)
+endif()
+
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Werror")
+if(NOT IS_CMAKE_31)
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Werror -std=c99")
```

Comment:
Sorry, I did not pay attention, I just copied the line and forgot to remote the duplicates. Thank you.